test/sanity_test.sh

```diff
@@ -363,6 +364,8 @@ run_zip_test_v2()
 	sw_dfl_hw_ifl /tmp/syslog
 	hw_dfl_sw_ifl /tmp/syslog
 	hw_dfl_hw_ifl /tmp/syslog
+	export WD_COMP_EPOLL_EN=0
+	hw_dfl_hw_ifl /tmp/syslog
```

Comment:
If you want to change the environment variable, it's better not to define it as global variable.
```
WD_COMP_EPOLL_EN=1 hw_dfl_hw_ifl /tmp/syslog
WD_COMP_EPOLL_EN=0 hw_dfl_hw_ifl /tmp/syslog
```

Reply:
It is modified now, please review again.